docs/pages/billing/usage-based-pricing.mdx

```diff
@@ -0,0 +1,160 @@
+
+For example, consider an account subscribed to the Production plan which does 15 medium Android builds, and 10 large iOS builds in a billing cycle:
+
+| Description             | Price | Quantity | Total |
+| ----------------------- | ----: | -------: | ----: |
+| Android builds (medium) |    $1 |       15 |   $15 |
+| iOS builds (large)      |    $4 |       10 |   $40 |
+| EAS Build Credit        |       |          |  -$55 |
+| **Total**               |       |          |    $0 |
+
+Since the credit is included in the Production plan, the subscriber pays $0 for their 25 builds.
+
+Next, consider another example where the credit limit is exceeded:
+
+| Description             | Price | Quantity | Total |
+| ----------------------- | ----: | -------: | ----: |
+| Android builds (medium) |    $1 |       20 |   $20 |
+| Android builds (large)  |    $2 |       10 |   $20 |
+| iOS builds (medium)     |    $2 |       15 |   $30 |
+| iOS builds (large)      |    $4 |       15 |   $60 |
+| EAS Build Credit        |       |          |  -$99 |
+| **Total**               |       |          |   $31 |
+
+In this scenario, the subscriber pays $31 for 60 builds instead of $130 because the EAS Build Credit covers $99.
+
+<BoxLink
+  title="Pricing schedule"
+  description="Visit our pricing page for more information on the pricing schedule for supported build platforms and the available resource classes."
+  href="https://expo.dev/pricing"
+  Icon={CreditCard02Icon}
+/>
+
+### EAS Update
+
+Usage-based pricing for EAS Update comprises two metrics monthly updated users and global edge bandwidth.
+
+Monthly updated users reflect the number of unique users who download an update in a billing cycle. At the same time, global edge bandwidth represents the total amount of bandwidth used for all update downloads. Moreover, each monthly updated user includes 40 MiB of global edge bandwidth, up to that plan's global edge bandwidth limit.
+
+> **Note**: A monthly updated user counts only once per billing cycle, regardless of how many updates this user downloads.
+
+Each plan has a number of monthly updated users and global edge bandwidth included as part of the subscription. These differ for each plan and the most updated numbers. See our [Pricing page](https://expo.dev/pricing), for more information.
+
+> **Note**: The [On-demand plan](/billing/plans/#on-demand) includes the same amount of monthly updated users and global edge bandwidth as the Free plan.
+
+For example, let's consider a subscriber to the On-demand plan who deploys 5 updates of 20 MiB each via EAS Update to 10,000 users. The subscription to the plan includes 1,000 monthly updated users and 100 GiB per month. As a result, the subscriber's overage bill will be:
```

Comment:
```suggestion
For example, let's consider a subscriber to the On-demand plan who deploys 5 updates of 20 MiB each via EAS Update to 10,000 users. The subscription to the plan includes 1,000 monthly updated users and 100 GiB per month. As a result, the subscriber's bill for extra usage will be:
```

Reply:
I would also change the examples on this page to use 20 updates that are 5 MiB each. 20 MiB is on the larger side.

Reply:
Update the example.